notebooks/community/model_evaluation/automl_tabular_classification_model_evaluation.ipynb

```diff
@@ -0,0 +1,1429 @@
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "52d622c274d2"
+      },
+      "outputs": [],
+      "source": [
+        "PIPELINE_ROOT = f\"{BUCKET_URI}/pipeline_root/pet_adoption_{UUID}\"\n",
+        "parameters = {\n",
+        "    \"project\": PROJECT_ID,\n",
+        "    \"location\": REGION,\n",
+        "    \"root_dir\": PIPELINE_ROOT,\n",
+        "    \"model_name\": model.resource_name,\n",
+        "    \"target_column_name\": \"Adopted\",\n",
+        "    \"batch_predict_gcs_source_uris\": [DATA_SOURCE],\n",
+        "    \"batch_predict_instances_format\": \"csv\",\n",
+        "    \"batch_predict_explanation_data_sample_size\": 3000,\n",
+        "}"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "0409b0f330c2"
+      },
+      "source": [
+        "Create a Vertex AI pipeline job using the following parameters:\n",
+        "\n",
+        "- `display_name`: The name of the pipeline, this will show up in the Google Cloud console.\n",
+        "- `template_path`: The path of PipelineJob or PipelineSpec JSON or YAML file. It can be a local path, a Google Cloud Storage URI or an Artifact Registry URI.\n",
+        "- `parameter_values`: The mapping from runtime parameter names to its values that\n",
+        "        control the pipeline run.\n",
+        "- `enable_caching`: Whether to turn on caching for the run.\n",
+        "\n",
+        "Learn more about the `PipelineJob` class from [this documentation](https://cloud.google.com/python/docs/reference/aiplatform/latest/google.cloud.aiplatform.PipelineJob).\n",
+        "\n",
+        "After creating, run the pipeline job using the configured `SERVICE_ACCOUNT`."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "894afe1ba396"
+      },
+      "outputs": [],
+      "source": [
+        "job = aiplatform.PipelineJob(\n",
+        "    display_name=PIPELINE_DISPLAY_NAME,\n",
+        "    template_path=\"tabular_classification_pipeline.json\",\n",
+        "    parameter_values=parameters,\n",
+        "    enable_caching=True,\n",
+        ")\n",
+        "\n",
+        "job.run(service_account=SERVICE_ACCOUNT)"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "ce6beLsXASnK"
+      },
+      "source": [
+        "## Model Evaluation"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "mKRTDi8ioXBY"
+      },
+      "source": [
+        "In the results from last step, click on the generated link to see your run in the Cloud Console.\n",
+        "\n",
+        "In the UI, many of the pipeline DAG nodes expand or collapse when you click on them. Here is a partially-expanded view of the DAG (click image to see larger version).\n",
```

Comment:
can you spell out this acronym? i.e. "directed acyclic graph (DAG)"

Reply:
done.